addons/l10n_tr/models/account_move_line.py

```diff
@@ -0,0 +1,22 @@
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+class AccountMoveLine(models.Model):
+    _inherit = 'account.move.line'
+
+    l10n_tr_exception_code_id = fields.Many2one(comodel_name='l10n_tr.exception_reason')
+    l10n_tr_exception_code_domain_ids = fields.Many2many(comodel_name='l10n_tr.exception_reason', store=True,
```

Comment:
1) there is no need to store this `m2m`.
2) `available_l10n_tr_exception_code_ids` instead ?
3) Also:
```suggestion
    l10n_tr_exception_code_domain_ids = fields.Many2many(
        comodel_name='l10n_tr.exception_reason',
        compute='_compute_l10n_tr_exception_code_domain_ids',
    )
```
=> better management of free space on the left

Reply:
Good name, but I want to keep `l10n_tr` prefix, so I used `l10n_tr_available_exception_code_ids`